webview/__init__.py

```diff
@@ -96,24 +91,39 @@ def _create_children(other_windows):
     guilib = initialize(gui)
 
     for window in windows:
-        window._initialize(guilib, multiprocessing, http_server)
+        window._initialize(guilib, _multiprocessing, http_server)
 
     if len(windows) > 1:
-        t = Thread(target=_create_children, args=(windows[1:],))
+        t = threading.Thread(target=_create_children, args=(windows[1:],))
         t.start()
 
     if func:
         if args is not None:
             if not hasattr(args, '__iter__'):
                 args = (args,)
-            t = Thread(target=func, args=args)
+            t = threading.Thread(target=func, args=args)
         else:
-            t = Thread(target=func)
+            t = threading.Thread(target=func)
         t.start()
 
     guilib.create_window(windows[0])
 
 
+def start(*args,**kwargs):
+    block = kwargs.get('block', True)
+    daemon = kwargs.pop('daemon', True)
+    
+    if block:
+        real_start(*args, **kwargs)
+        
+    else:
+        mp.get_context('fork')
+        p=mp.Process(target=real_start, args=args, kwargs=kwargs)
+        p.daemon = daemon 
```

Comment:
daemon would be always True. It should be verified whether multiprocessing module is able to shutdown spawned GUI process automatically or whether additional quite logic is needed.

Reply:
> daemon would be always True.
if daemon is True python kills gui loop automatically
**to avoid this users shod use something like this**
```python
 ...
webview.start(daemon = True)
print('free main thread')
while True:
    pass # keep gui loop
```
so i added both `block` and `daemon` args

Reply:
Sorry, daemon would be False by default, so it would block the main thread.
Join or other way (http server loop or something like that) would be require to prevent program termination. I see no value in letting implicitly child process be alive. 

By the way WkWebView on macOS uses multiprocessing architecture by default. It must be verified whether it works with daemonized multiprocessing. If not, then a custom solution to terminate gui loop is needed.